frontend/src/components/activePage/Workspaces/WorkspacesTable.tsx

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+
+import { FC, useState } from 'react';
+import { Table } from 'antd';
+import { CaretRightOutlined } from '@ant-design/icons';
+import TemplatesTable from '../Templates/TemplatesTable';
+import RowHeading from '../RowHeading/RowHeading';
+import { IWorkspace } from '../ActiveLanding/ActiveLanding';
+
+export interface IWorkspacesTableProps {
+  workspaces: Array<IWorkspace>;
+  destroyAll: () => void;
+}
+
+const WorkspacesTable: FC<IWorkspacesTableProps> = ({ ...props }) => {
+  const { workspaces, destroyAll } = props;
+  const { Column } = Table;
+  const [expandedID, setExpandedID] = useState(['']);
+
+  const accordion = (expanded: boolean, record: IWorkspace) => {
+    expanded ? setExpandedID([record.key]) : setExpandedID(['']);
+  };
+  return (
+    <Table
+      dataSource={workspaces}
+      pagination={false}
+      showHeader={false}
+      expandable={{
+        expandedRowKeys: expandedID,
+        // eslint-disable-next-line react/no-multi-comp
+        expandIcon: ({ expanded, onExpand, record }) => (
+          <CaretRightOutlined
+            onClick={e => onExpand(record, e)}
+            rotate={expanded ? 90 : 0}
+          />
+        ),
+        // eslint-disable-next-line react/no-multi-comp
+        expandedRowRender: record => (
+          <TemplatesTable
+            templates={record.templates}
+            destroyAll={destroyAll}
+          />
+        ),
+        expandRowByClick: true,
+        indentSize: 100,
```

Comment:
I don't think it is strictly related to this line. But check the storybook on mobile, because the workspaces/templates sections become very small and the `...` menu gets pushed on a new line

Reply:
As far as the mobile view is concerned I haven't really focused on that screen size as this frontend is Desktop oriented and the mobile view is just too narrow. I've focused around the `lg` and `md` breakpoints thinking of when a user splits his screen between the Crownlabs site and another window.
Regardless, I want to add ellipses to all the template and workspace titles which should mitigate the problem also on the narrowest of views

Also I should remove the `indentSize` line as it does not work at all.